festflow/templates/festflow/keynote_view.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+	<title>{{ keynote.name }}</title>
```

Comment:
follow something uniform in all `html` files, you're using 4 spaces here, and all other files follow 2 spaces